frontend/models/product-list/index.ts

```diff
@@ -58,14 +64,16 @@ export async function findProductList(
       metaDescription: productList.metaDescription ?? null,
       filters: productList.filters ?? null,
       image:
-         productListImageAttributes == null
-            ? null
-            : getImageFromStrapiImage(productListImageAttributes, 'large'),
+         productListImageAttributes != null
+            ? getImageFromStrapiImage(productListImageAttributes, 'large')
+            : deviceWiki
+            ? getDeviceImage(deviceWiki)
+            : null,
       ancestors: createProductListAncestors(productList.parent),
       // Strapi sort order is case sensitive, so we need to improve on it in memory
       children: sortProductListChildren(
          filterNullableItems(
-            productList.children?.data.map(createProductListChild)
+            productList.children?.data.map(createProductListChild(deviceWiki))
```

Comment:
Curious why this is a higher-order function instead of
```ts
            productList.children?.data.map(child => createProductListChild(child, deviceWiki))
```

Not that I think my version's better, just curious about the style choice.

Reply:
Not a specific reason for it.. I've just opted for the first refactor that I had in mind.. Maybe I just liked the idea of not having an arrow function there 😅 If you want I can refactor it as you're proposing, just let me know 👍

Reply:
Nope; totally fine the way it is. Thanks for satisfying my curiosity!